### ISSUE TITLE
default date range

### ISSUE THREAD

**Issue by sandikbarr** [PRX/metrics.prx.org#33]
When first loading the app, a default date range should be selected that matches one of the canned date ranges. (Maybe Two weeks?)

**Comment by gcampo88** [comment]
might vote for 1 week so they can see the 15 min option for interval?

**Comment by sandikbarr** [comment]
For 10 out of the 14 days of '2 weeks' the 15 minute option is visible, maybe we could relax Castle to 14 days for 15 min instead of the 10 day limitation. I dunno; I've also gotten input that 15 minutes shouldn't be available at all, ever (internal, not from user testing.) The reason I did not choose 'This week' is because the chart looks dumb when it's just one or two days of daily data points (but perhaps in that case, it should show hourly rather than daily?) I changed it from the rolling 14 days because I got feedback that a 'When' option should be selected. I don't think it's all quite nailed down yet, but the purpose of this ticket was to address that specific feedback about having a 'When' option selected.
